control-station/src/components/SensorBoxes/SensorBoxContainer.tsx

```diff
@@ -0,0 +1,11 @@
+import CCcontainer from "./Camera&Console/CCcontainer"
+import SensorContainer from "./Sensors/SensorsContainer"
+import "./style.css"
+export default function SensorBoxContainer(){
```

Comment:
Suggestion: the name `SensorBoxContainer` is a bit verbose, consider `Sensors` or `SensorData`